Tasks/ServiceFabricUpdateAppVersions/Update-ApplicationVersions.ps1

```diff
@@ -29,10 +29,13 @@ try {
 
     $logAllChanges = ((Get-VstsInput -Name logAllChanges -Require) -eq "true")
 
-    $versionSuffix = Get-VstsInput -Name versionSuffix -Require
-    Write-Host (Get-VstsLocString -Key VersionSuffixLabel -ArgumentList $versionSuffix)
+    $replaceVersion = ((Get-VstsInput -Name versionBehavior -Require) -eq "Replace")
```

Comment:
Yes it has a default in 'task.json' and from my testing it would just use the default.

However,  I removed the '-require' just to be safe.  If 'Get-VstsInput -Name versionBehavior' returns nothing, this line would result in '$replaceVersion = $false', which is the same as the default.